apple-nav/src/App.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Link, Route } from 'react-router-dom';
+import { AnimatedSwitch } from 'react-router-transition';
+import data from './assets/dummyData';
+import Links from './components/Header/Links';
+import SubLink from './components/Header/SubLink'
+import Page from './components/Page/Page';
+import Home from './components/Home/Home';
+import './App.scss';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      links: data,
+      sub: [],
+      subName: ''
+    }
+  }
+
+  subLinkCheck = (sub, subName) => {
+    this.setState({
+      sub: sub === undefined || null ? [] : sub,
```

Comment:
In JS both `undefined` and `null` are evaluated to `falsy`, so you can make this line cleaner by doing:
```
  sub: sub ? sub : [],
```